packages/hub/indexing/card-utils.ts

```diff
@@ -1,6 +1,19 @@
+import Error from '@cardstack/plugin-utils/error';
 import { todo } from '@cardstack/plugin-utils/todo-any';
 import { SingleResourceDoc, ResourceObject, ResourceIdentifierObject, CollectionResourceDoc } from "jsonapi-typescript";
-import { get, uniqBy } from 'lodash';
+import { get, uniqBy, isEqual, sortBy } from 'lodash';
+import logger from '@cardstack/logger';
+import { join } from "path";
+import {
+  ensureDirSync,
+  pathExistsSync,
+  writeFileSync,
+  removeSync,
+} from "fs-extra";
+
+const cardsDir: string = join(process.cwd(), 'card_modules');
```

Comment:
If we can use system temp instead, that would avoid allowing cards to "accidentally" resolve dependencies from the app. Also, it means less things to put into `.gitignore` et al.

Reply:
sounds good!